packages/owl-bot/src/constants.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export const OWL_BOT_POST_PROCESSOR_COMMIT_MESSAGE_MATCHER =
```

Comment:
I was worried that between the `\n\n` and the unicode :owl:, there was a risk that the message returned from the GitHub API wouldn't match the full `OWL_BOT_POST_PROCESSOR_COMMIT_MESSAGE`.

I'm extracted a substring for matching commit messages which we can be sure is unique to the post-processor, but doesn't have special characters.